test/unit/vl-map-wmts-layer.test.html

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+
+  <script type="module" src="../../src/vl-map-all.js"></script>
+  <script src="/node_modules/sinon/pkg/sinon.js"></script>
+</head>
+
+<body>
+  <test-fixture id="vl-map-wmts-layer-fixture">
+    <template>
+      <vl-map>
+        <vl-map-wmts-layer data-vl-url="https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmts" data-vl-layer="grb_sel" data-vl-name="GRB Wegenkaart"
+      data-vl-min-resolution="2" data-vl-max-resolution="4"></vl-map-wmts-layer>
+      </vl-map>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="vl-map-wmts-layer-fixture-no-resolutions">
+    <template>
+      <vl-map>
+        <vl-map-wmts-layer data-vl-url="https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmts" data-vl-layer="grb_sel" data-vl-name="GRB Wegenkaart">
+        </vl-map-wmts-layer>
+      </vl-map>
+    </template>
+  </test-fixture>
```

Comment:
deze mag weg denk ik. was een probeersel toen het nog niet duidelijk was wat ik met die resoluties zou gaan doen.
voorlopig default alleen dus.

Reply:
Die wordt precies wel gebruikt in een test: `testVisibilityAtResolution(() => fixture('vl-map-wmts-layer-fixture-no-resolutions'));`?

Reply:
Ah ja, nee sorry dan klopt. Ik was met iets anders aan t verwarren. Laat maar staan.